### PR TITLE
JSON light client config

### PR DESCRIPTION
**Pull request by 7AC**

### CODE REVIEW

File: beacon-chain/lightclient/config.go [hunk 1]
```diff
@@ -36,3 +58,54 @@ func NewConfig(chainConfig *params.BeaconChainConfig) *Config {
 		SecondsPerSlot:               chainConfig.SecondsPerSlot,
 	}
 }
+
+func NewConfigFromJSON(config *ConfigJSON) (*Config, error) {
```
Comment (qinlz2) [inline beacon-chain/lightclient/config.go:133]: This is fine, but maybe we can use `json.Unmarshaler` interface?  (Or added a wrapper to call NewConfigFromJSON)
 
 So we can use standard json interface (and will be good for nested decoding automatically)

Comment (7AC) [reply beacon-chain/lightclient/config.go:133]: mm how would unmarshaler help here? We're producing a different kind of object

Comment (7AC) [reply beacon-chain/lightclient/config.go:133]: are you suggesting we marshal to ConfigJSON -> bytes and unmarshal the other direction?

Comment (7AC) [reply beacon-chain/lightclient/config.go:133]: check my next commit let me know if that's what you meant

**Review by qinlz2 — APPROVED** [review]

**Review by qinlz2 — APPROVED** [review]

**Pull request merged**